spring-cloud-commons/src/main/java/org/springframework/cloud/client/loadbalancer/reactive/RetryableLoadBalancerExchangeFilterFunction.java

```diff
@@ -87,6 +90,7 @@ public RetryableLoadBalancerExchangeFilterFunction(LoadBalancerRetryPolicy retry
 		this(retryPolicy, loadBalancerFactory, properties, Collections.emptyList());
 	}
 
+	@Deprecated
```

Comment:
Please add javadoc with `@deprecated` tag indicating the suggested replacement.

Reply:
Added